src/nvim/api/ui_events.in.h

```diff
@@ -0,0 +1,70 @@
+#ifndef NVIM_API_UI_EVENTS_IN_H
+#define NVIM_API_UI_EVENTS_IN_H
+
+// This file is not compiled, just parsed for definitons
+#ifdef INCLUDE_GENERATED_DECLARATIONS
+#error "don't include this file, include nvim/ui.h"
+#endif
+
+#include "nvim/api/private/defs.h"
+#include "nvim/func_attr.h"
+#include "nvim/ui.h"
+void resize(Integer rows, Integer columns)
+    FUNC_API_SINCE(2);
+void clear(void)
+    FUNC_API_SINCE(2);
+void eol_clear(void)
+    FUNC_API_SINCE(2);
+void cursor_goto(Integer row, Integer col)
+    FUNC_API_SINCE(2);
+void mode_info_set(Boolean enabled, Array cursor_styles)
+    FUNC_API_SINCE(2);
+void update_menu(void)
+    FUNC_API_SINCE(2);
+void busy_start(void)
+    FUNC_API_SINCE(2);
+void busy_stop(void)
+    FUNC_API_SINCE(2);
+void mouse_on(void)
+    FUNC_API_SINCE(2);
+void mouse_off(void)
+    FUNC_API_SINCE(2);
+void mode_change(String mode, Integer mode_idx)
+    FUNC_API_SINCE(2);
+void set_scroll_region(Integer top, Integer bot, Integer left, Integer right)
+    FUNC_API_SINCE(2);
+void scroll(Integer count)
```

Comment:
Since `ui_append_lines` and `ui_delete_lines` are gone, a comment here would help.

```c
/// Positive `count` scrolls down (deletes screen lines).
/// Negative `count` scrolls up (appends screen lines).
void scroll(Integer count)
```